### DISCUSSION TITLE
Using the header for just the first page

### DISCUSSION THREAD

**Discussion by shimonmagal** [gotenberg/gotenberg#432] [Q&A]
Howdy,
I am trying to use the header (with an image), but have it limited to just the first page (very common use case of doing a letterhead for just the first page). But couldn't find anything in the documentation that can enable that.

I tried using CSS selectors such as:
```
@page: first {
 img: { display: block}
}
```

Or use pagedjs_first_page but nothing seems to work.

Using the latest 7.5 version of course...

Thank you

**Comment by gulien** [comment]
Hello @shimonmagal,

Best option would be to add the image directly in the HTML of the first page. You cannot use the `Header/Footer` feature as it applies to all page and there is nothing in the Google API to apply them only on the first page.
